### PR TITLE
Standardisation

### PR DESCRIPTION
**Pull request by iprafols**
This commit contains a merge from master with all the conflicts resolved

### CODE REVIEW

File: bin/picca_pk2fits.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env python
 
 from astropy.io import fits
+import numpy as np
 import scipy as sp
```
Comment (julianbautista) [inline bin/picca_pk2fits.py:5]: there is no more need of scipy ?

File: py/picca/fitter/parameters.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import numpy as np
 import scipy as sp
```
Comment (julianbautista) [inline py/picca/fitter/parameters.py:2]: scipy

File: bin/picca_pk2fits.py (hunk 1 above)
Comment (iprafols) [reply bin/picca_pk2fits.py]: updated

File: py/picca/fitter/parameters.py (hunk 2 above)
Comment (iprafols) [reply py/picca/fitter/parameters.py]: updates

**Comment by iprafols** [comment]
I rerun the tests on the commit 165d123 which originally passed to double check that there is no problem with versions. Not sure why travis tests now fail

I fixed commit ad4ce1c, sp.interpolate was changed to np.interpolate but this is not in numpy

**Comment by iprafols** [comment]
With the new fix the tests pass, so I think we can go ahead and merge

**Review by julianbautista — APPROVED** [review]

**Pull request merged**